### PR TITLE
[REV] website_sale: open configurator with only no variants attributes

### PR DESCRIPTION
**Pull request by chevalierv**
This reverts commit 436c1c6.

The commit was fixing an issue where the product configurator dialog would not be shown for custom attribute, no variant, and dynamic attribute using the quick buy add button on the '/shop' page.

The commit introduces an undesirable behavior: the product configurator would always be open if customers choose none of the multi-checkbox options.

### CODE REVIEW

File: addons/website_sale_product_configurator/tests/test_website_sale_configurator.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
+# -*- coding: utf-8 -*-
```
Comment (Feyensv) [inline addons/website_sale_product_configurator/tests/test_website_sale_configurator.py:1]: don't bring back the utf 8, it's causing the red ci-style :D

File: addons/website_sale_product_configurator/views/templates.xml [hunk 2]
```diff
@@ -77,7 +77,7 @@
                         <t t-if="product and not combination">
                             <t t-set="combination" t-value="product_template._get_first_possible_combination()"/>
                         </t>
-                        <t t-if="not already_configured" t-call="website_sale.variants">
```
Comment (Feyensv) [inline addons/website_sale_product_configurator/views/templates.xml:80]: Did you check it's not behaving strangely for people having the patched version and not updating their module?

Comment (chevalierv) [reply addons/website_sale_product_configurator/views/templates.xml:80]: I just tested it right now and it looks good to me.

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**